dist/tools/prep/prep.py

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2015  Hauke Petersen <dev@haukepetersen.de>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import sys
+import os
+from os import path
+import argparse
+import yaml
+import templates
```

Comment:
you might also want to import
```
from __future__ import print_function
```
in your python files so that the `print()` function is also available for python (>=) 2.6